### PR TITLE
Please review Mobile aboutsection to merge into main

### PR DESCRIPTION
**Pull request by flov3rh**
Create a webpage from scratch following template number 3, with no linter errors showing

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,5 +35,146 @@ <h1>Howdy! I am Flover. <span> I am a software developer</span></h1>
         <a href="#middle" id="down-button" class="fa-solid fa-chevron-down"></a>
       </header>
     </section>
+    <section id="middle">
+      <h2>My Recent Works</h3>
+      <div class="container">
+        <article class="grid-item">
```
Comment (Alusp) [inline index.html]: - [x] Good job so Far :clap: , however, please ensure that your class name: `grid-item` is referenced correctly in your `styles.css` file and kindly use `flex` properties to align the contents of your card items. as part of the requirement that says: `In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.` please resolve it. :slightly_smiling_face:

File: css/styles.css [hunk 2]
```diff
@@ -59,29 +162,58 @@ a {
...
   display: block;
 }
 
+button {
+  background-color: #36b37f;
+  color: #fff;
+  border-radius: 5px;
+  padding: 10px;
+  margin: 15px auto;
+}
```
Comment (Alusp) [inline css/styles.css]: - [x] Good job so far, :clap: However, Please ensure that your button design matches the given design as part of the requirement that says: `You must stick to the design as much as possible (e.g., font, colors, images, text, margins)` using this figma [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&mode=design&t=rPm74ST0zW6b1ScP-0) matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently.  Please ensure that you pay close attention the design  :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Accepted type</td>
    </tr>
    <tr>
      <td>

S btn

</td>
      <td>

F btn

</td>
    </tr>
  </table>



- [x] And kindly follow the button styling: `Active, Hover and Pressed`

H

File: css/styles.css [hunk 3]
```diff
@@ -14,19 +123,21 @@
...
-  justify-content: space-around;
   margin: auto;
   padding: 0;
-  width: 30%;
+}
+
+.list-grid {
+  display: grid;
+  grid-template: auto / repeat(9, 1fr);
 }
```
Comment (Alusp) [inline css/styles.css]: Good job so far, However, Please ensure that your list of technologies design matches the given design as part of the requirement that says: `You must stick to the design as much as possible (e.g., font, colors, images, text, margins)` using this figma [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&mode=design&t=rPm74ST0zW6b1ScP-0) matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently.  Please ensure that you pay close attention the design  :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

F Tlist

</td>
      <td>

F tlist

</td>
    </tr>
  </table>

File: css/styles.css [hunk 4]
```diff
@@ -59,29 +162,58 @@ a {
...
+  border-radius: 5px;
+  padding: 10px;
+  margin: 15px auto;
+}
+
 @media only screen and (max-width: 760px) {
+  #about-me > ul {
+    display: grid;
+    grid-template-columns: 1fr;
+  }
```
Comment (Alusp) [inline css/styles.css]: - [x] As part of the project requirement, you are required to use `flexbox` to align the contents of the `about-me` Please resolve that. :slightly_smiling_face: 
- [ ] Your design also is not matching the given design, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently.  Please ensure that you pay close attention the design  :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

F desi

</td>
      <td>

figma Des

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @flov3rh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

 ## Highlights

- [x] Linters set-up ✔️
- [x] Commit messages descriptive ✔️
- [x] Professional PR Summary :heavy_check_mark:  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 2 above)
Comment (flov3rh) [reply css/styles.css]: Added :active effect, however, I do not see any difference between enabled and on hover

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @flov3rh 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Matching the Figma design ✔️
- Proper use of semantic tags ✔️
- Well-structured files ✅

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**